catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py

```diff
@@ -0,0 +1,824 @@
+
+class RedirectDnsAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    redirect_dns: RedirectDns = Field(default=None, validation_alias="redirectDns", serialization_alias="redirectDns")
+
+
+class AppqoeOptimizationAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    appqoe_optimization: AppqoeOptimization = Field(
+        default=None, validation_alias="appqoeOptimization", serialization_alias="appqoeOptimization"
+    )
+
+
+class LossCorrectionAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    loss_correction: LossCorrection = Field(
+        default=None, validation_alias="lossCorrection", serialization_alias="lossCorrection"
+    )
+
+
+class CountAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    count: Global[str] = Field(default=None)
+
+
+class LogAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    log: Global[bool] = Field(default=None)
+
+
+class CloudSaasAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    cloud_saas: Global[bool] = Field(default=None, validation_alias="cloudSaas", serialization_alias="cloudSaas")
+
+
+class CloudProbeAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    cloud_probe: Global[bool] = Field(default=None, validation_alias="cloudProbe", serialization_alias="cloudProbe")
+
+
+class CflowdAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    cflowd: Global[bool] = Field(default=None)
+
+
+class NatPoolAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    nat_pool: Global[int] = Field(default=None, validation_alias="natPool", serialization_alias="natPool")
+
+
+class NatAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    nat: Nat = Field(default=None)
+
+
+class SigAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    sig: Global[bool] = Field(default=None)
+
+
+class FallbackToRoutingAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    fallback_to_routing: Global[bool] = Field(
+        default=None, validation_alias="fallbackToRouting", serialization_alias="fallbackToRouting"
+    )
+
+
+class SseAction(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    sse: Sse = Field(default=None)
+
+
+Actions = Union[
+    SlaClassAction,
+    BackupSlaPreferredColorAction,
+    SetAction,
+    RedirectDnsAction,
+    AppqoeOptimizationAction,
+    LossCorrectionAction,
+    CountAction,
+    LogAction,
+    CloudSaasAction,
+    CloudProbeAction,
+    CflowdAction,
+    NatPoolAction,
+    NatAction,
+    SigAction,
+    FallbackToRoutingAction,
+    SseAction,
+]
+
+
+class Sequences(BaseModel):
```

Comment:
please add helper methods so user can add items sequentially using builtin types (no additional imports)
check example:
https://github.com/cisco-open/cisco-catalyst-wan-sdk/blob/dev-uxmt/catalystwan/models/configuration/feature_profile/sdwan/topology/custom_control.py#L202
This greatly simplifies item creation process for user:
https://github.com/cisco-open/cisco-catalyst-wan-sdk/blob/dev-uxmt/catalystwan/integration_tests/feature_profile/sdwan/topology/test_topology.py#L62

Reply:
Done